### ISSUE TITLE
bug: missing icons after updating to nerd fonts 3.0

### ISSUE THREAD

**Issue by petobens** [stevearc/aerial.nvim#250]
### Neovim version (nvim -v)

Latest nightly

### Operating system/version

Arch Linux

### Output of :AerialInfo

-

### Describe the bug

I've updated today to use nerd fonts 3.0 and there are some missing glyphs/icons. 

### Steps To Reproduce

You can see in the screenshot that there are no icons for function or class:
![image](https://user-images.githubusercontent.com/2583971/235459770-69bc0a7b-758e-44a6-bf64-80f83d42fb1e.png)

### Expected Behavior

Have those icons

**Comment by petobens** [comment]
Using https://github.com/onsails/lspkind.nvim/pull/64 seems to solve the problem. So I guess we'll have to wait for that to get merged.

**Comment by stevearc** [comment]
Yeah, this is a lspkind.nvim issue. I've already updated all the icons in my own plugins. Will leave this issue open for tracking.
